snap/info.go

```diff
@@ -553,6 +553,29 @@ type TimerInfo struct {
 	Timer string
 }
 
+// StopModeType is the type for the "stop-mode:" of a snap app
+type StopModeType string
+
+func (st StopModeType) KillAll() bool {
```

Comment:
This feels oddly named, what is this even supposed to mean? Please document it at least.